### ISSUE TITLE
Clarify use of "Dataview source filter" or add to the settings

### ISSUE THREAD

**Issue by RobColes** [ivan-lednev/obsidian-day-planner#320]
Love the plug-in, Thank you!

Documentation says:
"Dataview source filter
You can set up Dataview to search for tasks only in specific folders or tags. This improves performance on large vaults...."

Which is great - I have separate daily notes for work and home and this would let me target them.

I couldn't find where this should be specified in the UI for Day Planner (or DataView).

Looking in the data.json for day-planner, I can see a "dataviewSource" element, but whenever I try to add anything here, it breaks the plugin and where the plugin should display just says "Plugin no longer active".

If I do a dataview query:
```dataview
TASK from "Daily"
```
then that selects the tasks I want.
Trying to add that to the dataviewSource element as:
```
 "dataviewSource": "TASK from \"Daily\" ",
```
(the double quotes around daily are escaped with a backslash)
just gives the "Plugin no longer active" message.

Would it be possible to clarify the use in the docs, and ideally add this to the settings UI?

Thanks

**Comment by ivan-lednev** [comment]
Hey, I'll update the docs. In the meantime, here's where the source is configured:
![image](https://github.com/ivan-lednev/obsidian-day-planner/assets/41428836/7c1b5eea-2128-4cd4-966c-32f383524b9d)

In your case, just add `"Daily"`.

**Comment by RobColes** [comment]
Thanks - that worked perfectly. I have the planner pinned in a narrow panel on the right hand side of the screen and the funnel icon isn't visible, not that I would probably have thought to try it anyway.

**Comment by Jafner** [comment]
Hey I'm not sure if I'm misunderstanding, misconfiguring, or misusing the source filter function and would really appreciate some insight.  

I have all my daily notes in a `/Daily` folder. In that folder I also have a `TODO` note which contains a list of tasks without due dates.  

This is my first time working with Dataview. I created a new note with a DQL codeblock view like:  

```txt
 TASK
 FROM "Daily"
```

Which returns a huge list of all tasks from all notes in the "Daily" folder (including the TODO note). As expected (if a bit unrefined).  

However, no matter what I enter, the only tasks I can see in the Day Planner Timeline are those created within Daily notes since 2024-05-16 (the date I set up Day Planner and Tasks).  I've tried `-"Daily"` and `"Daily/TODO"` to explicitly include the folder/file. I've tried giving the `#TODO` tag to the Daily/TODO note and using the `-#TODO` / `-"#TODO"` filters. Nothing seems to cause tasks from that note to be included in the planner.  

I tried explicitly assigning a date to one of the tasks (via `📅 2024-05-18` syntax from Tasks plugin, and via `2024-05-18 18:00` syntax from Day Planner plugin). It did not show up in the timeline. 

I tried adding a dataview block inside the daily note which pulled the tasks from the TODO note. This did not change the behavior.

I feel like I'm completely misunderstanding some fundamental concept here.
